### DISCUSSION TITLE
Podcast page

### DISCUSSION THREAD

**Discussion by entured** [AzuraCast/AzuraCast#6710] [General]
If we go to my main Podcasts page we see what goes wrong: https://radios.soyoutuber.com/public/2/podcasts
The issue is that it seems to center the entire podcast list on the screen and those above the top cannot be seen. because?

**Comment by BusterNeece** [comment]
@entured We found and fixed this issue in the latest Rolling Release version; I'd recommend updating to that to get the fixes applied to your podcasts page.

**Comment by entured** [comment]
♥️
